migrations/versions/5988f77859e7_.py

```diff
@@ -0,0 +1,31 @@
+"""Add relationship between diagnosis and biospecimen
```

Comment:
Can you add version here? I think it's `1.0.2`

Reply:
Should I wait until you merge read groups #352 in? If so, will it then be 1.1.1?